src/locales/en-US/list.ftl

```diff
@@ -200,11 +200,18 @@ ensure-logins-checked = Ensure the “Logins” checkbox is selected in sync pre
 setting-to-allow-sync = In order to allow your logins to be synced to other devices, this setting must be checked. <openprefs>Open sync preferences</openprefs></p>
 
 banner-promote-device =
-  .title = { -product-full-name }
-  .details = Download our mobile app to take your passwords with you everywhere.
-  .actionLabel = Learn More
+  .title = Take your passwords everywhere
+  .details = download our app for iOS or Android:
```

Comment:
How is this used exactly? Why is this lowercase?

Reply:
It's lowercase to match the spec:

<img width="1272" alt="Screen Shot 2019-05-08 at 8 15 57 AM" src="https://user-images.githubusercontent.com/96396/57386607-8d70de80-7169-11e9-88f8-a10455f8be99.png">

Reply:
OK, that's far from great. You're splitting a sentence in two, and nobody will ever know if you don't tell. Also, that dash might not work with all locales.

Can we get a single message for the whole sentence here?

Reply:
Sure. How should I organize the strings so that I can apply an emphasis style to just the initial part of the sentence?

Reply:
That's a technical that I'm not sure how to answer. Your best bet would be @pike, with Stas out.

Can you use text-level elements like `<strong>`? Or use another tag to enclose it (I see you have a `<go>` somewhere else).

```
.title = <strong>Take your passwords everywhere</strong> – download our app for iOS or Android:
```

Reply:
You can pass a strong element with class as a prop, see https://github.com/projectfluent/fluent.js/wiki/React-Overlays.